src/ui/Facet/ValueElementRenderer.ts

```diff
@@ -181,7 +181,7 @@ export class ValueElementRenderer {
   }
 
   private initAndAppendLabel() {
-    this.label = $$('label', { className: 'coveo-facet-value-label' }).el;
```

Comment:
I expect there are custom CSS rules out there assuming a label. Wouldn't the change to a div break those styles? Why is it necessary to change the tag vs. adding a role?

Reply:
That's a good point, I'll add a role instead.